development/releasing.md

```diff
@@ -10,7 +10,14 @@ When all the required issues are fixed in master it's time to put together a rel
 ```bash
 npm install --save libphonenumber@latest
 ```
-  - Update the git submodules (api and sentinel) to the latest and commit to `master` and push.
+  - Confirm the the git submodules (api and sentinel) are on the latest commit on the right branch (should mirror the parent repo).
+```bash
+git submodule update --remote api
+git submodule update --remote sentinel
+git status --exit-code api sentinel || \
+git commit -m 'bumping submodules' api sentinel && \
```

Comment:
interesting, didn't know you could include filenames here.  Would it also work:
```
git commit -m 'bumping submodules' -- api sentinel
```
?